src/elements/dialogs/session-details.html

```diff
@@ -96,7 +100,7 @@ <h3>{$ sessionDetails.speakers $}</h3>
                       hidden$="[[speaker.country]] == 'Unknown'">[[speaker.country]]</span></div>
                 </div>
               </div>
-            </div>
+            </button>
```

Comment:
Anything clickable/tappable should be an interactive element such as a button. https://www.w3.org/WAI/WCAG21/Techniques/general/G90